web/pages/docs/guides/pytorch-lightning-integration.mdx

```diff
@@ -0,0 +1,117 @@
+import DocsLayout from "../../../layouts/docs";
+
+<DocsLayout title="Pytorch Lightning integration">
+
+
+_This is an experimental feature, and the API may change in the future._
+
+Replicate works with any machine learning framework, but it includes a callback that makes it easier to use with [Pytorch Lightning](https://www.pytorchlightning.ai/).
+
+`ReplicateCallback` behaves like [Pytorch Lightning's `ModelCheckpoint` callback](https://pytorch-lightning.readthedocs.io/en/stable/generated/pytorch_lightning.callbacks.ModelCheckpoint.html), but in addition to exporting a model at the end of each epoch, it also:
+
+1. Calls `replicate.init()` at the start of training to create an experiment, and
+2. Calls `experiment.checkpoint()` after saving the model at [`on_validation_end`](https://pytorch-lightning.readthedocs.io/en/stable/callbacks.html#on-validation-end). If no validation is defined, then the checkpoint is saved at [`on_epoch_end`](https://pytorch-lightning.readthedocs.io/en/stable/callbacks.html#on-epoch-end). All metrics that have been logged during training with [`self.log()`](https://pytorch-lightning.readthedocs.io/en/latest/logging.html#automatic-logging) are saved to the Replicate checkpoint.
+
+Here is a simple example:
+
+```python
+import torch
+from torch.nn import functional as F
+from pytorch_lightning.core.lightning import LightningModule
+from pytorch_lightning import Trainer
+from torch.optim import Adam
+from torch.utils.data import DataLoader, random_split, Subset
+from torchvision.datasets import MNIST
+from torchvision import transforms
+
+from replicate.pl_callback import ReplicateCallback
```

Comment:
Shall we add #highlight-line and #highlight-start and end similar to the Keras guide. It helps show how little extra code Replicate adds.